frontend/public/logo.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="314" height="252" viewBox="0 0 314 252"><text transform="translate(1 220)" fill="#0967d2" font-size="249" font-family="FontAwesome6Pro-Regular, 'Font Awesome \36  Pro'"><tspan x="0" y="0">scale-unbalanced</tspan></text></svg>
```

Comment:
Das SVG ist mit der Schriftart gemacht? Das würde ich ändern in ein pure svg ohne schriftart? Geht das auf geräten ohne Font Awesome Font?

Reply:
So wie es jetzt ist perfekt! :)